cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -265,15 +272,14 @@ def start_kube_dns(sdn_plugin):
         hookenv.log('kube-apiserver not ready, not requesting dns deployment')
         return
 
+    message = 'Rendering the Kubernetes DNS files.'
```

Comment:
This message was originally in the `launch_dns` function which I removed in refactoring. Just preserving the message, it's not a new one!